static/app/views/performance/http/httpSamplesPanel.tsx

```diff
@@ -86,6 +90,16 @@ export function HTTPSamplesPanel() {
     });
   };
 
+  const handleResponseCodeClassChange = newResponseCodeClass => {
+    router.replace({
+      pathname: location.pathname,
+      query: {
+        ...location.query,
+        responseCodeClass: newResponseCodeClass.value,
```

Comment:
I guess this means that this affects the header as well

<img width="730" alt="image" src="https://github.com/getsentry/sentry/assets/18689448/f9436ec2-b4cc-4ba4-be85-64709b6d4512">

Do you think it makes sense to keep that?

Reply:
No, that's an oversight, fixed!